### ISSUE TITLE
SA1629 reports false-positives for author tags

### ISSUE THREAD

**Issue by x3ntrix** [DotNetAnalyzers/StyleCopAnalyzers#2721]
We have encountered false-positives for the author tags in our code after updating to 1.1.0-beta007.

```csharp
/// <summary>
///     Gets or sets the name.
/// </summary>
/// <author> john.doe@example.com </author>
public string Name { get; set; }
```

The warning says that the documentation (in this case the mail address) should end with a period.

**Comment by sharwell** [comment]
@vweijsters @pdelvo The underlying issue here is which top-level elements to analyze. We currently have a hard-coded exception for `seealso`. It seems like the following are reasonable approaches:

1. Use a hard-coded whitelist for elements to analyze
2. Use a hard-coded blacklist for elements to *not* analyze
3. Use a new configuration property to make (1) customizable
4. Use a new configuration property to make (2) customizable

Currently we implement 2, but my initial preferences are either 1 or 4.

**Comment by vweijsters** [comment]
I would prefer option 4 with `seealso` being the default value, as I think it would result in the least amount of discussion / maintenance. Any hard-coded list is arbitrary and will probably have people asking about additional inclusions / exclusions. An overly broad hard-coded exclusion list would reduce the effectiveness of the rule.
